### PR TITLE
Tests for CASSANDRA-12403

### PR DESCRIPTION
**Pull request by stef1927**
Some new tests for the slow query log, introduced by [CASSANDRA-12403](https://issues.apache.org/jira/browse/CASSANDRA-12403).

### CODE REVIEW

File: cql_tests.py [hunk 1]
```diff
@@ -785,6 +785,192 @@ def materialized_view_test(self):
...
+
+        debug(session.execute(SimpleStatement("SELECT * from test1 where id = 1",
+                                              consistency_level=ConsistencyLevel.ONE,
+                                              retry_policy=FallthroughRetryPolicy())))
+
+        debug(session.execute(SimpleStatement("SELECT * from test1 where id = 1",
+                                              consistency_level=ConsistencyLevel.ONE,
+                                              retry_policy=FallthroughRetryPolicy())))
+
+        debug(session.execute(SimpleStatement("SELECT * from test1 where token(id) > 0",
```
Comment (ptnapoleon) [inline cql_tests.py]: Why are we wrapping these all in debug? What do we expect to see that wouldn't be printed by the python driver's logger?

File: cql_tests.py [hunk 2]
```diff
@@ -785,6 +785,192 @@ def materialized_view_test(self):
...
+                                              retry_policy=FallthroughRetryPolicy())))
+
+        time.sleep(1)  # do our best to ensure logs had a chance to appear
+
+        self._check_logs(node, "operations were slow", 'system.log', 0)
+        self._check_logs(node, "SELECT \* FROM ks.test2", 'debug.log', 0)
+
+    def _check_logs(self, node, pattern, filename, num_expected):
+        ret = node.grep_log(pattern, filename=filename)
+        self.assertEquals(num_expected, len(ret),
```
Comment (ptnapoleon) [inline cql_tests.py]: You could just use `assert_length_equal` from `assertions.py` here

File: cql_tests.py (hunk 1 above)
Comment (stef1927) [inline cql_tests.py]: I removed the debug wrappers. How do we enable the driver's logger?

File: cql_tests.py (hunk 2 above)
Comment (stef1927) [inline cql_tests.py]: Replaced, thanks.

File: cql_tests.py (hunk 1 above)
Comment (ptnapoleon) [inline cql_tests.py]:
```
import logging
# set python-driver log level to INFO by default for dtest
logging.getLogger('cassandra').setLevel(logging.INFO)
```

is in `dtest.py`. You could just set that to `logging.DEBUG` I believe, for the duration of this test

**Comment by ptnapoleon** [comment]
Left the driver logging reply in-line, after that +1, thanks Stefania!

**Comment by stef1927** [comment]
Thanks Philip, I'm merging the PR as is without changing the driver log level. We don't need the driver to log at DEBUG level for this test, I merely wanted to know for my own benefit. The intent of the debug statements was to show the query results but that's not strictly necessary for this test.

**Pull request merged**